### PR TITLE
Add "next" to fix next is not defined error

### PR DESCRIPTION
**Pull request by DmitriyMolch**
While investigating the "Unable to retrieve header and parent from supplied hash" issue found that the `next` function is missing in the API function declaration which caused the app to crash and alert in the PROD alerts channel.

### CODE REVIEW

File: api/src/services/healthService.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ async function checkBlockchainHealth(req, res, next) {
   }
 }
 
-async function checkBlockchainBlocksFinalization(req, res) {
+async function checkBlockchainBlocksFinalization(req, res, next) {
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:132]: Add README

File: api/src/services/healthService.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ async function checkBlockchainBlocksFinalization(req, res) {
   }
 }
 
-async function checkBlockchainBlocksProducing(req, res) {
+async function checkBlockchainBlocksProducing(req, res, next) {
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:170]: Please, confirm we don't have this issue in other places

Comment (DmitriyMolch) [reply api/src/services/healthService.js:170]: Checked. There is no such problem in other API.

File: api/src/services/healthService.js (hunk 1 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js:132]: Added

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**